platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/EventIntake.java

```diff
@@ -63,42 +67,47 @@ public class EventIntake {
     /** Stores events, expires them, provides event lookup methods */
     private final ShadowGraph shadowGraph;
 
+    private final PreconsensusEventStreamSequencer sequencer = new PreconsensusEventStreamSequencer();
+
+    /**
+     * Writes preconsensus events to disk.
+     */
+    private final PreConsensusEventWriter preConsensusEventWriter;
```

Comment:
I've made this change so that I can move forward on this PR (I really don't like the implementation of `EventObserverDispatcher`). 

We really need to reach consensus on a design for inter-component communication... I feel like each time we write new things like the preconsensus event stream there is severe pain when it comes to wiring them together.